python/MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+# C++ files
+recursive-include rmm *.hpp
```

Comment:
I don't see what paths this is supposed to be including. The headers are in `include/rmm` from the root of the repo, and there is are no `.hpp` files in `python/rmm`.

Reply:
It's in `python/`. These paths are relative to that

Reply:
Gotcha that makes more sense. Yeah noticed the same. Guessing this is holdover from when there were some `.hpp` files (and suppose there could always be some again)

That all being said, agree there is some follow up work here